azimuth/config.py

```diff
@@ -222,6 +222,9 @@ class CommonFieldsConfig(ProjectConfig, extra=Extra.ignore):
     # Will use CUDA and will need GPUs if set to True.
     # If "auto" we check if CUDA is available.
     use_cuda: Union[Literal["auto"], bool] = "auto"
+    # Memory of the dask cluster. Regular is 6GB, Large is 12GB.
+    # For bigger models, large might be needed.
+    large_dask_cluster: bool = False
```

Comment:
My only interrogation is if we should generalize the config to a string and let the user choose whatever size.
```suggestion
    large_dask_cluster: string = "12GB"
```

Reply:
I'm not sure - @Dref360 set that up initally.

Reply:
6GB fit well for a Mac and 12GB is enough to run most models

Reply:
I would keep it as if for now, until this no longer works.